### ISSUE TITLE
b.cesm3_cam041_mom.B1850MOM.ne30_L58_t061.005

### ISSUE THREAD

**Issue by cecilehannay** [NCAR/amwg_dev#114]
**Description:**
Same as #111 + bugfix for the sea-ice to fix excessive fresh water (See [discussion #109](https://github.com/NCAR/amwg_dev/discussions/109)) 

**Case directory:**
Locally (if still available):
/glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam041_mom.B1850MOM.ne30_L58_t061.005

On github:
https://github.com/NCAR/amwg_dev/tree/b.cesm3_cam041_mom.B1850MOM.ne30_L58_t061.005

**Sandbox:**
Locally (if still available):
/glade/work/hannay/cesm_tags/cesm3_cam6_3_041_MOM3

On github:
https://github.com/NCAR/amwg_dev/tree/cesm3_cam6_3_041_MOM3
hash: [5a56ad3](https://github.com/NCAR/amwg_dev/commit/5a56ad39c5ef4f26695cb49632f26437b57d3e76)

**Diagnostics:**
AMWG diags (if available)
[https://webext.cgd.ucar.edu/B1850MOM/b.cesm3_cam041_mom.B1850MOM.ne30_L58_t061.005/atm/](https://webext.cgd.ucar.edu/B1850MOM/b.cesm3_cam041_mom.B1850MOM.ne30_L58_t061.005/atm/)

**Contacts:**
@cecilehannay and @gustavo-marques

**Comment by gustavo-marques** [comment]
The run did not start because of the following error
`ERROR: FLDLST: 1 errors found, see log`

**Comment by cecilehannay** [comment]
After looking at the namelists, we noticed differences (see below) between:
- ``B1850MOM`` +  add ``-chem waccm_sc_mam4`` in CAM_CONF_OPTS
- ``BWsc1850MOM`` (This compset doesn't exist so we 1850_CAM60%WCSC_CLM50%BGC-CROP_CICE_MOM6_MOSART_CISM2%GRIS-NOEVOLVE_SWAV_SESP_BGC%BDRD) 

It seems we don't get all the forcing with adding just adding ``-chem waccm_sc_mam4`` in CAM_CONF_OPTS -> we decided to use  ``BWsc1850MOM`` for the next coupled run.

This issue will be closed and replaced by #115

```
< /glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam041_mom.B1850MOM.ne30_L58_t061.005/CaseDocs/atm_in
> /glade/p/cesmdata/cseg/runs/cesm2_0/b.cesm3_cam041_mom.B1850WcMOM.ne30_L58_t061.005/CaseDocs/atm_in

<  ch4vmr               =    791.6e-9  
<  co2vmr               =    284.7e-6  
<  f11vmr               =    12.48e-12  
<  f12vmr               =    0.0        
<  flbc_list            = ' '
<  n2ovmr               =    275.68e-9  
>  flbc_cycle_yr                = 1850
>  flbc_file            = '/glade/p/cesmdata/cseg/inputdata/atm/waccm/lb/LBC_17500116-20150116_CMIP6_0p5degLat_c180905.nc'
>  flbc_list            = 'CO2','CH4','N2O','CFC11','CFC12','CFC11eq'
>  flbc_type            = 'CYCLICAL'
>  scenario_ghg         = 'CHEM_LBC_FILE'

<  solar_parms_data_file                = '/glade/p/cesmdata/cseg/inputdata/atm/cam/solar/SolarForcing1995-2005avg_c160929.nc'
>  solar_parms_data_file                = '/glade/p/cesmdata/cseg/inputdata/atm/cam/solar/SolarForcingCMIP6piControl_c160921.nc'


<  waccm_forcing_cycle_yr               = 2000
>  waccm_forcing_cycle_yr               = 1850

<  waccm_forcing_file           = 'SCWACCM_forcing_WACCM6_zm_5day_L70_1975-2014_c191121.nc'
>  waccm_forcing_file           = 'SCWACCM_forcing_zm_L70_CMIP6piControl001_MonthlyCycle1850_y21-50avg_c181012.nc'

```
